### PR TITLE
CI update to clang-8

### PR DESCRIPTION
**Pull request by anakrish**
Signed-off-by: Anand Krishnamoorthi <anakrish@microsoft.com>

### CODE REVIEW

**Comment by anakrish** [comment]
bors try

File: scripts/ansible/roles/linux/openenclave/vars/ubuntu/main.yml [hunk 1]
```diff
@@ -50,7 +50,7 @@ apt_arm_packages:
 
 validation_distribution_binaries:
   - "/usr/bin/shellcheck"
-  - "/usr/bin/clang-7"
+  - "/usr/bin/clang-8"
   - "/usr/bin/clang-8"
```
Comment (BRMcLaren) [inline scripts/ansible/roles/linux/openenclave/vars/ubuntu/main.yml:54]: You have a couple of double dependencies in the above files. Clang8 was added before, so only the lvi wrapper change is needed and then deprecating clang-7 which can be seen as optional.

**Review by radhikaj — APPROVED** [review]
Thanks @anakrish

File: scripts/ansible/roles/linux/openenclave/vars/ubuntu/main.yml [hunk 2]
```diff
@@ -8,11 +8,11 @@ shellcheck_target_version: "0.4.6"
 cmake_prefix: "/usr/local"
 
 #TODO: Flip to Clang-8 after https://github.com/openenclave/openenclave/pull/3574 is merged
-clang_binary_name: "clang-7"
+clang_binary_name: "clang-8"
 
 apt_packages:
   - "curl"
-  - "clang-7"
+  - "clang-8"
```
Comment (rs--) [inline scripts/ansible/roles/linux/openenclave/vars/ubuntu/main.yml:15]: `clang-8` is already listed on line 36

**Review by rs-- — APPROVED** [review]

File: scripts/ansible/roles/linux/openenclave/vars/ubuntu/main.yml (hunk 2 above)
Comment (anakrish) [reply scripts/ansible/roles/linux/openenclave/vars/ubuntu/main.yml:15]: Fixed.

**Comment by anakrish** [comment]
bors try

**Comment by anakrish** [comment]
bors try-

**Comment by anakrish** [comment]
bors try

File: scripts/ansible/roles/linux/openenclave/vars/ubuntu/main.yml (hunk 1 above)
Comment (BRMcLaren) [reply scripts/ansible/roles/linux/openenclave/vars/ubuntu/main.yml:52]: 51/52 are the same.

**Comment by rs--** [comment]
bors try

File: scripts/ansible/roles/linux/openenclave/vars/ubuntu/main.yml [hunk 3]
```diff
@@ -8,11 +8,11 @@ shellcheck_target_version: "0.4.6"
 cmake_prefix: "/usr/local"
 
 #TODO: Flip to Clang-8 after https://github.com/openenclave/openenclave/pull/3574 is merged
-clang_binary_name: "clang-7"
```
Comment (BRMcLaren) [inline scripts/ansible/roles/linux/openenclave/vars/ubuntu/main.yml:11]: On line 6 of this file, we need to change the clang_target_version to 8.0.0

**Comment by anakrish** [comment]
@BRMcLaren @rs-- 
Is this PR useful anymore or is this superceded by @rs-- 's work to update the images?

**Comment by BRMcLaren** [comment]
@anakrish Still needed :)

File: scripts/ansible/roles/linux/openenclave/vars/ubuntu/main.yml (hunk 3 above)
Comment (anakrish) [reply scripts/ansible/roles/linux/openenclave/vars/ubuntu/main.yml:11]: Done as suggested. Should the version be clang 8.0.1 or clang 8.0.0?
Not sure which version of clang CI was updated to
https://releases.llvm.org/download.html

Comment (BRMcLaren) [reply scripts/ansible/roles/linux/openenclave/vars/ubuntu/main.yml:11]: Thanks.

When I ran the validation locally, I saw 8.0.0

**Comment by mingweishih** [comment]
closing this for now. @rs-- will create a follow-up PR.

**Pull request closed without merging**